api/app_analytics/influxdb_wrapper.py

```diff
@@ -149,14 +154,20 @@ def get_event_list_for_organisation(organisation_id: int, date_range: str = "30d
         filters=f'|> filter(fn:(r) => r._measurement == "api_call") \
                   |> filter(fn: (r) => r["organisation_id"] == "{organisation_id}")',
         extra="|> aggregateWindow(every: 24h, fn: sum)",
-        date_range=date_range,
+        date_start=date_start,
+        date_stop=date_stop,
     )
     dataset = defaultdict(list)
     labels = []
     for result in results:
         for record in result.records:
             dataset[record["resource"]].append(record["_value"])
-            required_records = int(date_range[:-1]) + 1
+            if date_stop == "now()":
+                required_records = abs(int(date_start[:-1])) + 1
```

Comment:
Longer term comment: I think this is a good argument from factoring out the influx specific syntax and just using python types as per this issue: https://github.com/Flagsmith/flagsmith/issues/3417

Reply:
Well the one view that calls this code now transforms it to a negative signed string, so `now()` isn't technically possible. I think we should keep the refactoring work to #3417 since partially fixing it here will make things inconsistent. I'd rather have the whole codebase be upgraded at once.